core/web/router.go

```diff
@@ -258,15 +252,22 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		bdc := BulkDeletesController{app}
 		authv2.DELETE("/bulk_delete_runs", bdc.Delete)
 
-		ocrkc := OffChainReportingKeysController{app}
-		authv2.GET("/off_chain_reporting_keys", ocrkc.Index)
-		authv2.POST("/off_chain_reporting_keys", ocrkc.Create)
-		authv2.DELETE("/off_chain_reporting_keys/:keyID", ocrkc.Delete)
+		ekc := ETHKeysController{app}
+		authv2.GET("/keys/eth", ekc.Index)
+		authv2.POST("/keys/eth", ekc.Create)
+		authv2.DELETE("/keys/eth/:keyID", ekc.Delete)
+		authv2.POST("/keys/eth/import", ekc.Import)
+		authv2.POST("/keys/eth/export/:address", ekc.Export)
+
+		ocrkc := OCRKeysController{app}
+		authv2.GET("/keys/ocr", ocrkc.Index)
+		authv2.POST("/keys/ocr", ocrkc.Create)
+		authv2.DELETE("/keys/ocr/:keyID", ocrkc.Delete)
 
 		p2pkc := P2PKeysController{app}
-		authv2.GET("/p2p_keys", p2pkc.Index)
-		authv2.POST("/p2p_keys", p2pkc.Create)
-		authv2.DELETE("/p2p_keys/:keyID", p2pkc.Delete)
+		authv2.GET("/keys/p2p", p2pkc.Index)
+		authv2.POST("/keys/p2p", p2pkc.Create)
+		authv2.DELETE("/keys/p2p/:keyID", p2pkc.Delete)
```

Comment:
Does this router support namespacing?

Reply:
Yeah, but let's do it all at once in a separate PR